### DISCUSSION TITLE
Microchip doesn't appear to be storing OSCILLATOR ERROR values for 3224's?

### DISCUSSION THREAD

**Discussion by SpenceKonde** [SpenceKonde/megaTinyCore#559] [Q&A]
Kind of a bummer, I gotta say...

At least the 1624;s had them,. 

Anyone else playing with 2-series parts care to see what their sigrows look like? you can print the whole thing like so

```
volatile uint8_t * sigptr = (volatile uint8_t *) &SIGROW;
  sigptr = Serial.printHex(sigptr,16, ':');
  sigptr = Serial.printHex(sigptr,16, ':');
  sigptr = Serial.printHex(sigptr,16, ':');
  Serial.printHex(sigptr,16, ':');
```
output looks like 
1E:95:28:30:43:4D:4E:32:43:AE:6C:4B:12:FF _3F:0E_
_06_ FF:FF:FF:14 _05_ FF:FF:A2:05:1E:06:FF:FF:E0:F8
88:EC **FF:FF:FF:FF** FF:FF:AE:F4:E3:FC:1F:FF:FF:FF
FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:F1:BE

Bold text is where the osc error used to be. 2-series oscillators are so good that the values in there were like 0xFD (-3) to 0x03 (+3), and usually -1 to +1

Italics are values that vary between individual specimens and are not documented.  They vary over a very narrow range: the first one is always pretty close to 0x40 (nominally 64?) the next one is 0C, 0D or 0E on all parts I've seen, and the other two are always 05, 06 or 07. No strong correlations between any of them, 

Looks like they've decided their oscilators are good enough that they don;'t need to...?

**Comment by hmeijdam** [comment]
here you go, this comes from my 1626
```
1E:94:29:30:42:42:38:51:38:41:F8:27:44:FF:40:0E
06:FF:FF:FF:12:07:FF:FF:B6:04:34:04:FF:FF:E0:F8
8C:18:FF:FF:FF:FF:FF:FF:AE:F4:E3:FC:1F:FF:FF:FF
FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:8C:9B
```

and as a comparison this is from a 806
```
1E:93:24:32:50:32:51:4E:36:06:65:21:53:FF:42:0E
06:FF:FF:FF:10:07:1B:05:5B:05:9A:05:D9:05:EF:F8
95:02:04:07:FE:00:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF
FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF
```

and a 416
```
1E:92:21:51:50:51:50:4C:20:39:94:21:4A:FF:40:0E
06:FF:FF:FF:12:08:19:07:58:07:98:07:D7:07:EF:F8
93:01:FC:03:04:0A:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF
FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF:FF

```
